great_expectations/datasource/data_connector/runtime_data_connector.py

```diff
@@ -51,9 +54,80 @@ def __init__(
             batch_spec_passthrough=batch_spec_passthrough,
         )
 
-        self._batch_identifiers = batch_identifiers
+        if assets is None:
+            assets = {}
+        self._assets: Dict[str, Union[dict, Asset]] = assets
+        self._batch_identifiers: dict = {}
```

Comment:
@Shinnnyshinshin No type hints for object properties (convention).  Thanks.

Reply:
Thank you :)